src/braket/aws/aws_quantum_task.py

```diff
@@ -75,19 +74,20 @@
 
 class AwsQuantumTask(QuantumTask):
     """Amazon Braket implementation of a quantum task. A quantum task can be a circuit,
-    an OpenQASM program or an AHS program."""
+    an OpenQASM program or an AHS program.
+    """
 
     # TODO: Add API documentation that defines these states. Make it clear this is the contract.
-    NO_RESULT_TERMINAL_STATES = {"FAILED", "CANCELLED"}
-    RESULTS_READY_STATES = {"COMPLETED"}
+    NO_RESULT_TERMINAL_STATES: ClassVar = {"FAILED", "CANCELLED"}
+    RESULTS_READY_STATES: ClassVar = {"COMPLETED"}
     TERMINAL_STATES = RESULTS_READY_STATES.union(NO_RESULT_TERMINAL_STATES)
 
     DEFAULT_RESULTS_POLL_TIMEOUT = 432000
     DEFAULT_RESULTS_POLL_INTERVAL = 1
     RESULTS_FILENAME = "results.json"
 
     @staticmethod
-    def create(
+    def create(  # noqa: D417
```

Comment:
Is this for args and kwargs? I'd prefer we document these. Even to me it's unclear what these arguments are for. If we can't find any use cases or general reason why they are here, maybe we should consider removing them. If not, then surely we can find something to say about them

Reply:
I agree with this. In favor of onboarding the package to the new linting tools, I wanted to address these later as I think their docstrings are less obvious and need to be handled with more scrutiny. To get the builds to pass, I have opted to hide these from the linter but I have a follow up later on.

Reply:
Are we tracking these follow-ups somewhere?

Reply:
I can create a linting follow up task in asana to better capture these actions